tests/purs/warning/DeprecatedConstraintInForeignImport.out

```diff
@@ -0,0 +1,9 @@
+Error found:
```

Comment:
Do you know why this message says "Error" and not "warning"?

Reply:
Oh that is weird. Compiling the module with `stack exec purs -- compile` logs “Warning found:“ as expected 🤔

Reply:
This is because .out files are generated with `printErrorOrWarning`, which use `Language.PureScript.Errors.prettyPrintMultipleErrors` for both errors and warnings:
 https://github.com/purescript/purescript/blob/8869eb30cc4528a908e37cb97401938e59cbeffb/tests/TestCompiler.hs#L234

Reply:
This is fixed by https://github.com/purescript/purescript/pull/3846.

Reply:
I'd say let's merge #3846 first, and then you can update the golden test output in this PR.